docs/structured-streaming-kafka-integration.md

```diff
@@ -642,9 +642,9 @@ This way the application can be configured via Spark parameters and may not need
 configuration (Spark can use Kafka's dynamic JAAS configuration feature). For further information
 about delegation tokens, see [Kafka delegation token docs](http://kafka.apache.org/documentation/#security_delegation_token).
 
-The process is initiated by Spark's Kafka delegation token provider. When `spark.kafka.bootstrap.servers`,
+The process is initiated by Spark's Kafka delegation token provider. When `spark.kafka.bootstrap.servers` set,
 Spark considers the following log in options, in order of preference:
-- **JAAS login configuration**
+- **JAAS login configuration**, please see example below.
```

Comment:
Added this small pointer to make things more clear.

Reply:
Hi, @gaborgsomogyi . Which example is this pointing? Previous examples seems to be removed.

Reply:
@dongjoon-hyun down below with the same `JAAS login configuration` name.